dataikuapi/dss/managedfolder.py

```diff
@@ -108,6 +109,21 @@ def put_file(self, path, f):
                 "POST", "/projects/%s/managedfolders/%s/contents/%s" % (self.project_key, self.odb_id, utils.quote(path)),
                 "", f)
 
+    def upload_folder(self, path, folder):
+        """
+        Upload folder and its content as path in the managed folder.
```

Comment:
I think it would be good to document the detail of behavior that it puts "at the location", not "as child of location",

i.e. that upload_folder("target", "source") will result in "target" containing the content of "source", not in "target" containing "source"

Reply:
I rephrased into:

> Upload the content of a folder at path in the managed folder.
> 
> Note: upload_folder("target", "source") will result in "target" containing the content
> of "source", not in "target" containing "source".